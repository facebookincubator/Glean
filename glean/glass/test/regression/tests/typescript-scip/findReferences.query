action: findReferences
args: "test/ts/lsif/%60example.ts%60/Git%23"
