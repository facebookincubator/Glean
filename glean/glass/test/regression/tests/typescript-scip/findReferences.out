[
    "@generated",
    [
        {
            "filepath": "example.ts",
            "repository": "test",
            "span": {
                "length": 0,
                "start": 0
            }
        },
        {
            "filepath": "example.ts",
            "repository": "test",
            "span": {
                "length": 0,
                "start": 0
            }
        }
    ]
]